backend/src/caching/memcached.py

```diff
@@ -0,0 +1,41 @@
+from pymemcache.client import base
+
+from .cache import Cache
+
+
+class Memcached(Cache):
+    def __init__(self, config={}):
+        self.expire = config["expire"] if "expire" in config else 300
+        if "memcache_address" not in config:
+            raise ValueError("memcache_address not in config")
```

Comment:
I'm a little wary of 1) these error messages and 2) the fact that we're raising errors anyways; we want to, as they say, crash/fail gracefully. If something goes wrong with the cache, the entire backend could potentially go down.